src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -124,9 +127,18 @@ private async Task ProcessAsyncActions(AsyncAction[][] actions)
                             tasks.Add(this.context.CallSubOrchestratorWithRetryAsync(action.FunctionName, action.RetryOptions, action.InstanceId, action.Input));
                             break;
                         case AsyncActionType.CallEntity:
-                            var entityId = EntityId.GetEntityIdFromSchedulerId(action.InstanceId);
-                            tasks.Add(this.context.CallEntityAsync(entityId, action.EntityOperation, action.Input));
-                            break;
+                            {
```

Comment:
nit: I'm not necessarily opposed to using `{}`s in switch statements, but it currently doesn't match our existing code, so I would remove them.

Reply:
The `{}` are there because, otherwise, the compiler **complains** about having two variables `entityId` declared in the same scope, where the scope is the whole `switch`-statement. In other words, the `{}` help me define a new scope. 

I was certainly unsure how to handle this. I suppose I could declare `entityId` in the beginning of the switch and simply assign it in the `callEntity` and `signalEntity` cases. What do you think?

Reply:
I'm fine leaving it. Forgot about how variable context works in non-scoped switch statements.

The other option that would work would be to inline `EntityId.GetEntityIdFromSchedulerId(action.InstanceId)`, but I am not convinced that is any better than just leaving the scoping `{}`.

I wouldn't recommend declaring `entityId` before the switch statement, as it only really makes sense in two of the cases, and may even throw an error if a non-entity instance id is used.

Let's just leave it.